index.js

```diff
@@ -147,6 +147,24 @@ function setup(argv, env) {
     fs.writeFileSync(path.join(workingDir, cmdname), shim)
     fs.chmodSync(path.join(workingDir, cmdname), '0755')
   }
+  else if (cmdname === 'node') {
+    const nodePath = path.dirname(process.execPath)
+    const cmds = JSON.parse(env.NYC_CONFIG)._
```

Comment:
Although `nyc` is a primary user of `spawn-wrap` it's not the only user.  I tend to think it's not appropriate to look at `process.env.NYC_CONFIG`.  CC @bcoe @isaacs any thoughts on this?

Reply:
Perhaps instead of copying only the scripts from the process that `nyc` will use, it should copy all of them so any user of `spaw-wrap` and even chaining operations (one process calling the other) will work as expected.

Reply:
If running locally installed testers were a problem this could work, but it's not possible for us to copy every tester installed to the global path.  The truth is that manipulation of scripts as done in spawn-wrap is problematic, I fear it's not possible to eliminate 100% of edge cases.

FYI an alternative I'm working on is https://github.com/istanbuljs/nyc/pull/1169.  The idea is that this will use environmental variables to wrap child processes.  You can test this by running `npm install -D git://github.com/coreyfarrell/nyc#set-node-options`, then adding `--set-node-options=true` to your nyc command-line.  Keep in mind this has not passed review, is subject to change and once merged that branch will be deleted.  As mentioned in a comment on that PR a current issue is that clearing the environment will remove coverage from child processes.  I'm working on a solution to that but the PR isn't updated yet.